### DISCUSSION TITLE
Expo support?

### DISCUSSION THREAD

**Discussion by MiguelNiblock** [mmazzarolo/react-native-universal-monorepo#46] [Ideas]
Hi @mmazzarolo , 

Do you think your tool could be used by expo in this monorepo template? 

TLDR
I've been exploring several different ways to have an idyllic monorepo with react native and next with react-native-web, and the closest thing I've found so far is nx. However, their expo support is crappy at the moment (I tried their new expo generator, but it's full of errors. I currently am able to share react-native code between a nextjs and a vanilla react-native project using nx, but I would also like to have the option of an expo project in the same monorepo, also importing the shared code that goes into the other two projects. The value this would bring stems from the fact that expo provides a nicer development experience than vanilla react-native: For example, easily running app in devices over wifi, easier integration with nextjs (with their new adapter), the ability to publish a shareable link to your app in development, etc. And it would also be useful to compare the final bundle size of the app, built in bare vs expo, for the same app which comes from the same shared code.
